backend/api/src/main/kotlin/io/tolgee/api/v2/controllers/notification/NotificationOperationsController.kt

```diff
@@ -0,0 +1,30 @@
+package io.tolgee.api.v2.controllers.notification
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import io.tolgee.security.authentication.AllowApiAccess
+import io.tolgee.security.authentication.AuthenticationFacade
+import io.tolgee.service.notification.NotificationService
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@CrossOrigin(origins = ["*"])
+@RequestMapping(
+  value = [
+    "/v2/",
+  ],
+)
+@Tag(name = "Notification Operations", description = "Operations with notifications")
```

Comment:
You can keep the same tag, so it's not in different group in the UI.

```suggestion
@Tag(name = "Notifications", description = "Manipulates notifications")
```